### PR TITLE
Switch dockerhub to ghcr

### PR DESCRIPTION
**Pull request by WackerO**
This PR changes the container repo to ghcr

## PR checklist

- [x] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
  - [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/rnadeseq/tree/master/.github/CONTRIBUTING.md)
  - [ ] If necessary, also make a PR on the nf-core/rnadeseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [x] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by WackerO** [comment]
Question: Can I rename the push_dockerhub file or is the name used by some piece of code that I overlooked?

**Comment by ggabernet** [comment]
Hi Oskar, you can rename the GHA workflow, no problem

File: Dockerfile [hunk 1]
```diff
@@ -1,6 +1,8 @@
 FROM condaforge/mambaforge
-LABEL authors="Gisela Gabernet, Alexander Peltzer" \
-    description="Docker image containing all requirements for qbic-pipelines/rnadeseq pipeline"
+LABEL org.opencontainers.image.source=https://github.com/qbic-pipelines/rnadeseq
+LABEL org.opencontainers.image.description="Docker image containing all requirements for qbic-pipelines/rnadeseq pipeline"
+LABEL org.opencontainers.image.authors="Gisela Gabernet, Alexander Peltzer"
```
Comment (ggabernet) [inline Dockerfile:4]:
```suggestion
LABEL org.opencontainers.image.authors="Gisela Gabernet, Alexander Peltzer, Oskar Wacker"
```

**Comment by ggabernet** [comment]
The PR looks good to me, however you also need to specify `ghcr.io/qbicpipelines/rnadeseq:dev` inside the modules that use this container

File: .github/workflows/ci.yml [hunk 2]
```diff
@@ -36,13 +36,13 @@ jobs:
             environment.yml
       - name: Build new docker image
         if: env.MATCHED_FILES
-        run: docker build --no-cache . -t qbicpipelines/rnadeseq:dev
+        run: docker build --no-cache . -t ghcr.io/qbic-pipelines/rnadeseq:dev
```
Comment (SusiJo) [inline .github/workflows/ci.yml:39]: Is it `ghcr.io/qbic-pipelines`?

File: .github/workflows/push_github_container_reg.yml [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    name: Push new Docker image to Docker Hub
+    runs-on: ubuntu-latest
+    # Only run for the qbicpipelines repo, for releases and merged PRs
+    if: ${{ github.repository == 'qbic-pipelines/rnadeseq' }}
+    steps:
+      - name: Check out pipeline code
+        uses: actions/checkout@v2
+
+      - name: Build new docker image
+        run: docker build --no-cache . -t ghcr.io/qbicpipelines/rnadeseq:latest
```
Comment (SusiJo) [inline .github/workflows/push_github_container_reg.yml:22]: Or `ghcr.io/qbicpipelines`?

File: modules/local/report.nf [hunk 4]
```diff
@@ -1,6 +1,6 @@
 process REPORT {
 
-    container 'qbicpipelines/rnadeseq:dev'
+    container 'ghcr.io/qbic-pipelines/rnadeseq:dev'
```
Comment (SusiJo) [inline modules/local/report.nf:3]: Not sure now which one is right
```suggestion
    container 'ghcr.io/qbicpipelines/rnadeseq:dev'
```

**Comment by SusiJo** [comment]
Please make sure the repo prefix to which you push the container is consistent.

File: .github/workflows/ci.yml (hunk 2 above)
Comment (WackerO) [reply .github/workflows/ci.yml:39]: It's with a dash now :)

File: .github/workflows/push_github_container_reg.yml (hunk 3 above)
Comment (WackerO) [reply .github/workflows/push_github_container_reg.yml:22]: oops, that one is incorrect, well spotted!

File: modules/local/report.nf (hunk 4 above)
Comment (WackerO) [reply modules/local/report.nf:3]: this one's right already, with hypen

**Review by SusiJo — APPROVED** [review]
LGTM!

**Pull request merged**